.github/workflows/e2e-local.yml

```diff
@@ -0,0 +1,92 @@
+name: E2E tests / Locally
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install common tools
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: ./install-commons.sh
+
+      - name: Use Node.js 12
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-solidity-node-modules
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+          
+      - name: Install Go
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: |
+          sudo rm -rf /usr/local/go
+          ./install-go.sh
+          echo "Using Go: $(go version)"
+          echo "GOPATH=$HOME/go" >> $GITHUB_ENV
+          echo "$HOME/go/bin" >> $GITHUB_PATH
+          echo "/usr/local/go/bin" >> $GITHUB_PATH
```

Comment:
I'm not sure we need this both... Isn't it enough to add just `$HOME/go/bin` to GITHUB_PATH?